README.md

```diff
@@ -1,6 +1,10 @@
 # Budget_App
 
-> This is a budget app with backend built using Ruby on Rails framework. It involves creating  item categories and transactions from those categories. The categories have a more detailed page for the specific transaction.
+> This is a budget app with backend built using Ruby on Rails framework. It involves creating  item categories and transactions from those categories. The categories have a more detailed page for the specific transaction. this is a handy app for all your daily transactions
+
+## Live Version
+
+[Live](https://peaceful-mesa-18312.herokuapp.com)
 ## Built With
 
 - Ruby
```

Comment:
- [ ] The video presentation link is missing from the readme file
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).